modules/apps/frontend-taglib/frontend-taglib-clay/test.properties

```diff
@@ -11,24 +11,26 @@
     test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
         (testray.component.names == "Clay") OR \
         (\
```

Comment:
Extra parenthesis?

Reply:
Line 13, Line 35

Reply:
The previous expression is connected with OR, followed by AND, and the last two expressions are connected with AND to form a group. However, due to the previous SF, the parentheses have been removed.
AND and OR appearing at the same level can affect readability and are prone to errors